basis/common-router/src/main/java/com/softgroup/common/router/TokenKey.java

```diff
@@ -0,0 +1,13 @@
+package com.softgroup.common.router;
+
+import org.jose4j.keys.AesKey;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by alex on 26.02.17.
+ */
+
+@Component
+public class TokenKey {
```

Comment:
@Alexey007gold remove this. Token will be used in different way. I'll explain on lesson.